.github/workflows/verify-generate.yml

```diff
@@ -59,11 +59,6 @@ jobs:
     steps:
       - name: Check out code
         uses: actions/checkout@v2
-      - name: Install Go
```

Comment:
Same here. I thought at least for `make generate` it's required to explicitly install a go version in this workflow.

I thought the workflows which use `.github/actions/kamel-prepare-env` don't need explicit go installation but those which don't use it still require it.

Reply:
I removed by mistake also here. However the action just passed. Should I revert this change?

Reply:
I suspect it's because the default go installation in the ubuntu is used for the workflow. I think it's safer to revert it here as well.